CHANGELOG.md

```diff
@@ -41,6 +43,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Fixed the `InputCheckbox` and `InputCheckboxGroup` playwright controllers' `.expect_width()` to check the `width` property within the `style` attribute. (#1702)
 
+* Fixed a bug in `ui.conditional_panel()` that would cause the panel to repeatedly show/hide itself when the provided condition did not evaluate to a boolean value. (#1707)
+
+* Fixed a bug with `ui.input_slider()` when used as a range slider that made it impossible to change the slider value when both handles were at the maximum value. (#1707)
+
```

Comment:
I'm starting to think we should have a dedicated section of `shiny/README.md` dedicated to JS changes, so we can just point to that, but this is great for now, thanks.

Reply:
Agreed. I'd really like it if Shiny were a JavaScript package that we could just update directly rather than pulling from the R package.